submissions/Eugene-CG/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Eugene-CG/a-tiny-JS-world
+   Web app: https://eugene-cg.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const cat = {
+  species: "cat",
+  name: "Eugene",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Hey Fellas!",
+};
+const dog = {
+  species: "dog",
+  name: "Toby",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+const man = {
+  species: "man",
+  name: "John",
+  gender: "male",
+  legs: 2,
+  hands: 2,
+  saying: "I am a man",
+};
+const woman = {
+  species: "dog",
+  name: "Yennefer",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: "Nice to meet you",
+};
+const catWoman = {
+  species: "Dream",
+  name: "Noone",
+  gender: "female",
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
```

Comment:
Here you are only copying the initial value of the `cat.saying`. You have to figure out some way to connect those values, so that if you change `cat.saying`, `catWoman.saying` changes as well.